x-pack/plugins/enterprise_search/public/applications/enterprise_search/components/license_callout/license_callout.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useValues } from 'kea';
+
+import { EuiPanel, EuiFlexGroup, EuiFlexItem, EuiButton, EuiText } from '@elastic/eui';
+
+import { LicensingLogic } from '../../../shared/licensing';
+
+import { PRODUCT_SELECTOR_CALLOUT_HEADING } from '../../constants';
+
+import { LICENSE_CALLOUT_BODY, LICENSE_CALLOUT_BUTTON } from './constants';
+
+export const LicenseCallout: React.FC = () => {
+  const { hasPlatinumLicense, isTrial } = useValues(LicensingLogic);
+
+  if (hasPlatinumLicense && !isTrial) return null;
+
+  return (
+    <EuiPanel hasShadow={false} hasBorder className="productCard" paddingSize="l">
+      <EuiFlexGroup gutterSize="s" alignItems="center" justifyContent="spaceBetween">
+        <EuiFlexItem grow={7}>
+          <EuiText>
+            <h3>{PRODUCT_SELECTOR_CALLOUT_HEADING}</h3>
+          </EuiText>
+          <EuiText size="s">{LICENSE_CALLOUT_BODY}</EuiText>
+        </EuiFlexItem>
+        <EuiFlexItem grow={1}>
+          <div />
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <div>
+            <EuiButton href="/app/management/stack/license_management">
+              <EuiText size="s">{LICENSE_CALLOUT_BUTTON}</EuiText>
```

Comment:
Actually another quick question here - do we really need the `<EuiText size="s">`?  I checked and it looks like the text size isn't actually changing, it's just font weight that slightly adjusts... I might suggest losing this for simplicity honestly (and being more in-line with default EUI styles)

```suggestion
              {LICENSE_CALLOUT_BUTTON}
```

Reply:
Yep, not sure why I had that. I thought I remembered it making a difference.

Reply:
https://github.com/elastic/kibana/pull/99122/commits/b9588e44ceffc9ecb4a21fbaa6af69044f996d97